src/orion/benchmark/task/__init__.py

```diff
@@ -10,3 +10,6 @@
 from .rosenbrock import RosenBrock
 
 __all__ = ["BaseTask", "RosenBrock", "Branin", "CarromTable", "EggHolder"]
+
+# IDEA: Add an external entry_point for Tasks, so that we can add the profet and OpenML
```

Comment:
Yep! Can you please create an issue for this. Thanks! :)

Reply:
Done (#646 )

Reply:
Thanks! This comment should be removed then. :)